app/views/admin/action_pages/_index.html.erb

```diff
@@ -18,9 +18,9 @@
         <td><%= action_page.action_type %></td>
         <td><strong><%= link_to action_page.title, edit_admin_action_page_path(action_page) %></strong></td>
         <td class="nowrap"><%= action_page.created_at.strftime("%m-%d-%Y") %></td>
-        <td></td>
-        <td></td>
-        <td></td>
+        <td><%= action_page.event_summary[:views] %></td>
```

Comment:
this is a pretty awful N+1 query, but adding an `includes(:events)` to the controller query in `#filter_action_pages` is causing my rails server to lock up, then crash, locally. I'm investigating solutions now.

Reply:
going to add a couple of counter caches!